libs/librepcb/project/schematics/graphicsitems/sgi_symbol.cpp

```diff
@@ -66,7 +66,13 @@ void SGI_Symbol::updateCacheAndRepaint() noexcept
 {
     prepareGeometryChange();
 
+    // Docs: https://doc.qt.io/qt-5/qgraphicsitem.html#boundingRect
     mBoundingRect = QRectF();
+
+    // Docs: https://doc.qt.io/qt-5/qgraphicsitem.html#shape
+    // The default implementation calls boundingRect() to return a simple rectangular shape,
+    // but subclasses can reimplement this function to return a more accurate shape
+    // for non-rectangular items.
```

Comment:
Hmm honestly I don't like these comments, because:
- We have over a dozen graphics items, it's not practical to add these comments to each of them.
- I don't see a reason to copy Qt's documentation into our source code. When working on our graphics items, it should be clear anyway to read Qt's documentation (graphics items aren't our concept, it's the concept of Qt). See also their complete graphics view documentation: https://doc.qt.io/qt-5/graphicsview.html

Reply:
Well, they would have helped me a lot :) But I removed them.